samples/deployment/fabric-smart-client/the-simple-testing-network/README.md

```diff
@@ -16,10 +16,10 @@ make -C $FPC_PATH/ercc all docker
 
 Build the kv-test-go chaincode:
 ```bash
-export CC_NAME=kv-test-go
+export CC_NAME=fpc-kv-test-go
 export CC_PATH=$FPC_PATH/samples/chaincode/kv-test-go
 make -C $CC_PATH build docker
-export FPC_CHAINCODE_IMAGE=fpc/kv-test-go
+export FPC_CHAINCODE_IMAGE=fpc/fpc-kv-test-go
```

Comment:
Can't we keep it `FPC_CHAINCODE_IMAGE=fpc/kv-test-go`?

Reply:
No, the image name is created during build using CCID and adding `fpc/fpc-` ....

Reply:
well, we should have both ecc and ecc_go use the same naming convention right? When you build using ecc then it creates the image as `fpc/fpc-whatever`

Reply:
removing references to fpc- in names